test/awss3.jl

```diff
@@ -0,0 +1,47 @@
+function minio_server(body, dirs=[mktempdir()]; address="localhost:9005")
+    server = Minio.Server(dirs; address)
+
+    try
+        run(server; wait=false)
+        sleep(0.5)  # give the server just a bit of time, though it is amazingly fast to start
+
+        config = MinioConfig(
+            "http://$address"; username="minioadmin", password="minioadmin"
+        )
+        body(config)
+    finally
+        # Make sure we kill the server even if a test failed.
+        kill(server)
+    end
+end
+
+@testset "AWSS3 usage" begin
+    minio_server() do config
+        s3_create_bucket(config, "test-bucket")
+
+        file_format = "lpcm.zst"
```

Comment:
```suggestion
        file_format = "lpcm"
```

uncompressed is needed for byte-range to work

Reply:
oops, jinx ;)